src/wallet/wallet.h

```diff
@@ -160,7 +160,7 @@ class JSOutPoint
     // Transaction hash
     uint256 hash;
     // Index into CTransaction.vjoinsplit
-    size_t js;
+    uint64_t js;
```

Comment:
@str4d After pull request to your subtree is in, do you want me to create another PR?